content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+# Updating Bottlerocket
+
+This document covers the different ways to update Bottlerocket (for [ECS](#ecs), [Kubernetes](#kubernetes), and [SSM](#ssm)), as well as how to [lock your nodes to a specific release](#locking-to-a-specific-release).
+
+## Introduction
+
+Bottlerocket is designed to be updated in an [image-based fashion](https://github.com/bottlerocket-os/bottlerocket#updates).
+This means that updates are applied by replacing the entire image on disk, rather than applying a series of individual package updates.
+This is a departure from the traditional package-based Linux update model such as `apt` or `yum`, which are what you would find in Ubuntu or Amazon Linux.
+
+## Ways To Update
+
+There are a few ways to update Bottlerocket.
+The method you choose depends on the [Bottlerocket variant](https://github.com/bottlerocket-os/bottlerocket#variants) you are using and your environment.
+
+### ECS
+
+For the `aws-ecs-*` variants of Bottlerocket, we provide an [ECS updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#how-it-works).
+Please see the [ECS updater installation documentation](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#installation) for more information on how to install and use the ECS updater.
+
+Essentially, you will need to do the following:
+
+1. [Get subnet information](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#subnet-info) for your cluster:
+    - Get the ID of the VPC you want to use for the ECS updater.
+    - Get the Subnet ID from the VPC you found in the previous step (remember, the subnet must have internet access).
+2. [Get the name of the CloudWatch Logs log group](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#log-group) where the Bottlerocket ECS updater will send its logs.
+3. Get the name of the ECS cluster where you are running Bottlerocket container instances.
+4. [Install the Bottlerocket ECS updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#install) using a CloudFormation template (provided in the full ECS updater installation documentation, linked above).
+
+Detailed steps, including commands to run, are provided in the ECS updater documentation, linked above.
+
+**Note:** By default, the ECS updater checks for updates every 12 hours.
+
+### Kubernetes
+
+For the `aws-k8s-*` variants of Bottlerocket, there are a handful of possible ways to update Bottlerocket: Brupop, the EKS Console, `eksctl`, and SSM Command Documents.
+The following sections discuss each of these methods.
```

Comment:
Hmmm, different concepts are being conflated under the heading of "updating Bottlerocket"

- With the EKS console we are talking about replacing nodes with newer AMIs. I'm not sure that qualifies as "Upgrading Bottlerocket". There's nothing different about Bottlerocket here.
- eksctl? Another place that you can initiate node replacement I guess?
- brupop is software created to keep Bottlerocket nodes up-to-date using the Bottlerocket update system
- SSM Commands are a way to run the Bottlerocket update commands "yourself".

I'm not sure how to untangle all of this.

Reply:
Would renaming the doc "updating Bottlerocket clusters" (instead of individual nodes) be better?

That way, the node replacement methods make sense as cluster updates, and can be discussed alongside in-place upgrades.

Or, would it be clearer to separate out "in-place update" vs. "node replacement" methods?

Reply:
From a user's perspective, this page should answer a question along the lines of "Bottlerocket is out of date, how do I get it to a current version?" 

@mchaker Can you write an intro paragraph to disambiguate this difference?

Reply:
Yes. I'll write an intro paragraph describing two types of updates: in-place and node replacement, perhaps with a table grouping the different methods into their respective type.